pkgs/development/tools/dump_syms/default.nix

```diff
@@ -17,8 +17,13 @@
 let
   pname = "dump_syms";
   version = "2.2.2";
+
+  # Workaround for https://github.com/nixos/nixpkgs/issues/166205
+  workaround = lib.optionalAttrs stdenv.cc.isClang {
+    NIX_LDFLAGS = "-l${stdenv.cc.libcxx.cxxabi.libName}";
+  };
```

Comment:
Would there be a problem if `workaround` was replaced with the following inside the attribute set of `rustPlatform.BuildRustPackage`?

```nix
# Workaround for https://github.com/nixos/nixpkgs/issues/166205
env.NIX_LDFLAGS = lib.strings.optionalString stdenv.cc.isClang
  "-l${stdenv.cc.libcxx.cxxabi.libName}";

```

Reply:
I wasn't entirely sure if there was some implication on whether `NIX_LDFLAGS` is defined-but-empty vs not being defined at all so I erred on the side of cation (also as it stands right now, the Linux versions do not need to be rebuilt since the environment hasn't changed)

Reply:
I see. I do not have the knowledge to know if `NIX_LDFLAGS` as `""` may cause any different behavior than `NIX_LDFLAGS` unset.